Add possibility to configure mTLS `validityDays` and `renewalDays` for each `KafkaUser` (#12658)
strimzi/strimzi-kafka-operator: documentation/modules/appendix_crds.adoc

|type
|string
|Must be `tls`.
|validityDays
|integer
|Number of days for which the user certificate should be valid. If not configured, Clients CA configuration is used.
|renewalDays
|integer

> |Configures how many days before the certificate expiration should be the user certificate renewed. If not configured, Clients CA configuration is used.

Reviewer: ```suggestion
|Number of days before certificate expiration when the user certificate should be renewed. If not configured, Clients CA configuration is used.
```
